packages/client/src/App.js

```diff
@@ -60,6 +61,11 @@ const RouteList = () => {
       <Route path="/contact-us" element={<ContactUs />} />
       <Route path="/user-name" element="" />
       <Route path="/favourites" element={<FavouritePage />} />
+      <Route
```

Comment:
We should probably check if the user is admin, just like with the Inbox route. It's not really a security measure, but it prevents people from accidentally going to that site.